app/services/flow/base_step.rb

```diff
@@ -48,7 +48,7 @@ def create_response(form_submit_response, call_response)
     end
 
     def form_submit
-      FormResponse.new(success: true)
+      FormResponse.new(success: false)
```

Comment:
I see the commit describing this as "testing", I assume we probably don't want to change the default submission to be treated as a failure 😅

Reply:
For what it's worth, based on CI, the answer is poorly.